### PR TITLE
Added hackathons vidreview2 and curriculum to yaml

### PR DESCRIPTION
**Pull request by dtendr**
Forgot to pull request apparently

### CODE REVIEW

**Comment by dtendr** [comment]
Any reason why its been 19 days without a merge?

**Comment by ryansb** [comment]
Not that I can see. Ask @decause

File: scripts/people/2014/fall/dtendr.yaml [hunk 1]
```diff
@@ -12,8 +14,11 @@ hw:
   teamprop1: http://dtendr.blogspot.com/2014/09/team-proposal.html
   commarchreport: http://dtendr.blogspot.com/2014/09/commarch-report.html
   commarchpreso: https://docs.google.com/a/g.rit.edu/presentation/d/1ITde9mhXxQ5-e_xwCRvTBWqmZd-Du2RKeqO8r3iWcoQ/edit#slide=id.g3c4d4d6c3_30
+  curriculum: http://dtendr.blogspot.com/2014/10/ny-ma-4th-grade-math-curriculum.html
```
Comment (decause) [inline scripts/people/2014/fall/dtendr.yaml:17]: better late than never

:clock1130:

File: scripts/people/2014/fall/dtendr.yaml [hunk 2]
```diff
@@ -12,8 +14,11 @@ hw:
...
   teamprop2: http://dtendr.blogspot.com/2014/10/team-proposal-2.html
   vidreview1: http://dtendr.blogspot.com/2014/10/rip-remix-manifesto-video-review.html
   meetup2: http://dtendr.blogspot.com/2014/10/rocpy-second.html
   smoketest: http://dtendr.blogspot.com/2014/10/xo-smoke-test.html
-  vidreview2: http://dtendr.blogspot.com/2014/10/simple-made-easy.html
\ No newline at end of file
+  vidreview2: http://dtendr.blogspot.com/2014/10/simple-made-easy.html
+  litreview2: http://dtendr.blogspot.com/2014/11/hacking-politics-tldr-litreview.html
```
Comment (decause) [inline scripts/people/2014/fall/dtendr.yaml:23]: :clock1130:

Comment (decause) [inline scripts/people/2014/fall/dtendr.yaml:23]: :clock1130:

**Comment by decause** [comment]
:clock1130:

**Pull request merged**